nrf91x: Update quickstart with new ATT version (#414)
NordicSemiconductor/pc-nrfconnect-quickstart: Changelog.md

Reviewer: The new changelog entries will have to be added to a new section.
Should probably be v1.9.1, but @SlavaNordic can help with that

Reviewer 2: yes please create a new section. Version 1.9.1 will be fine. Also Tech writer should review. As this is a fork of the repo I'm unable to push to this branch to fix myself. Creating a separate PR in a separate repo does not sound right :)

> ## 1.9.1 - 2026-04-28

Reviewer 2: ```suggestion
## 1.9.1 - Unreleased
```

### Changed

- Asset Tracker Template firmware for the Nordic Thingy:91 X and the nRF9151 DK
  updated to v1.4.0.
- Link in the Asset Tracker evaluation step now points to the Asset Tracker
  Template "Connecting" documentation instead of the "Provisioning"

>   documentation.

Reviewer 3: ```suggestion
- Updated the Asset Tracker Template firmware for the Nordic Thingy:91 X and the nRF9151 DK
  to v1.4.0.
- Link in the Asset Tracker evaluation step now points to the Asset Tracker
  Template "Connecting" documentation instead of the "Provisioning"
  documentation.
```

- Asset Tracker app descriptions across all board configurations to include
  extra help text.
- Updated firmware and description for the nRF54LM20 DK.
- Asset Tracker Template firmware for the Nordic Thingy:91 X and the nRF9151

>   DK updated to v1.4.0 (with UART modem trace enabled).

Reviewer: We haven't mentioned whether UART modem trace is enabled in the changelog before. I also feel like it isn't necessary to mention this here.
```suggestion
- Asset Tracker Template firmware for the Nordic Thingy:91 X and the nRF9151
  DK updated to v1.4.0.
```
